contracts-cli/contract.js

```diff
@@ -0,0 +1,120 @@
+const { prompt } = require('inquirer')
+const { utils } = require('web3')
```

Comment:
Could use [web3-utils](https://www.npmjs.com/package/web3-utils) directly to avoid importing all of web3